calculator.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calculator</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="calculator.css" />
+  </head>
+  <body class="body">
+    <main>
+      <div class="calc">
+        <div class="calc-screen">
+          <p class="screen-values">1 + 1</p>
+          <div class="calc-result">
+            <p class="screen-equal">=</p>
+            <p class="screen-number">2</p>
```

Comment:
let's rename couple things
screen-values -> input
calc-result -> result-wrapper
screen-number -> result

don't forget rename them in CSS too